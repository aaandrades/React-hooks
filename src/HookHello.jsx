import React from 'react'

export const HookHello = () => {
    return (
        <div>
            Hello World
        </div>
    )
}
