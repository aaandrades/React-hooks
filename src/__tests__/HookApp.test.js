import React from "react";
import { shallow } from "enzyme";
import { HookHello } from "../HookHello";

describe("Testing in HookApp", () => {
  it("Should show correctly the component", () => {
    const wrapper = shallow(<HookHello />);
    expect(wrapper).toMatchSnapshot();
  });
});
