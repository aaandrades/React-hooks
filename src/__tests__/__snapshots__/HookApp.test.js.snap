// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in HookApp Should show correctly the component 1`] = `
<div>
  Hello World
</div>
`;
